### ISSUE TITLE
broken on OCaml 4.03.0

### ISSUE THREAD

**Issue by pelzlpj** [whitequark/ppx_include#1]
ppx_include is broken on OCaml 4.03.0 due to some tweaks in the parsetree typing. It looks like the fix is minor:

```
diff --git a/src/ppx_include.ml b/src/ppx_include.ml
index de83f82..1e8acac 100644
--- a/src/ppx_include.ml
+++ b/src/ppx_include.ml
@@ -12,7 +12,7 @@ let raise_errorf ?sub ?if_highlight ?loc message =
 let filename_of_payload ~loc payload =
   match payload with
   | PStr [{ pstr_desc = Pstr_eval (
-      { pexp_desc = Pexp_constant (Const_string (file, None)) }, _) }] ->
+      { pexp_desc = Pexp_constant (Pconst_string (file, None)) }, _) }] ->
     file
   | _ ->
     raise_errorf ~loc "[%%include]: invalid syntax"
```

**Comment by whitequark** [comment]
Yeah. Unfortunately this also needs cppo, as outlined here: https://github.com/whitequark/ppx_getenv/pull/3#issuecomment-216527902

**Comment by whitequark** [comment]
Published 1.1 on opam, which should be compatible with 4.02 and 4.03.
